pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -1810,55 +1810,50 @@ def test_dt64ser_sub_datetime_dtype(self):
     # TODO: This next block of tests came from tests.series.test_operators,
     # needs to be de-duplicated and parametrized over `box` classes
 
-    def test_operators_datetimelike_invalid(self, all_arithmetic_operators):
-        # these are all TypeEror ops
+    @pytest.mark.parametrize(
```

Comment:
I dont think this improves the clarity of this test.  id suggest something like:

```
def test_whatever(all_arithmetic_operators):
    op_str = all_arithmetic_operators
    if op_str in [add, sub, radd, rsub]:
        return # or skip, or use a different fixture; these are tested elsewhere

    tdser = [timedelta(minutes=5, seconds=3), NaT, timedelta(minutes=5, seconds=3)]
    dt64ser = [Timestamp("20111230"), Timestamp("20120101"), NaT]
    dt64tz_ser = ...

    for iter_over_pairs([tdser, dt64ser, dt64tz_ser]):
         with pytest.raises...
               op(whatever)

```

Reply:
Hmm at least with this version an error with any of the `dtlike-ser` won't prevent the other `dtlike-ser` from being tested due to the `parameterize`

Reply:
Fair enough.  I might be overly-wary of over-parametrizing in the arithmetic tests since there's a ton of them

Reply:
Hopefully with a second pass we can remove the need for this test entirely.